D&D_4E/D&D_4E.css

```diff
@@ -742,10 +742,10 @@
 }
 
 .sheet-rolltemplate-dnd4epower .sheet-monster {
-	background-color: -webkit-linear-gradient(left, darkgreen , chartreuse); /* For Safari 5.1 to 6.0 */
-    background-color: -o-linear-gradient(right, darkgreen, chartreuse); /* For Opera 11.1 to 12.0 */
-    background-color: -moz-linear-gradient(right, darkgreen, chartreuse); /* For Firefox 3.6 to 15 */
-    background-color: linear-gradient(to right, darkgreen , chartreuse); /* Standard syntax (must be last) */
+    background: -webkit-linear-gradient(left, darkgreen, chartreuse); /* For Safari 5.1 to 6.0 */
+    background: -o-linear-gradient(right, darkgreen, chartreuse); /* For Opera 11.1 to 12.0 */
+    background: -moz-linear-gradient(right, darkgreen, chartreuse); /* For Firefox 3.6 to 15 */
+    background: linear-gradient(to right, darkgreen , chartreuse); /* Standard syntax (must be last) */
```

Comment:
MDN page for reference, if you like: https://developer.mozilla.org/en-US/docs/Web/CSS/gradient/linear-gradient()